src/MountedTraversal.js

```diff
@@ -47,6 +47,9 @@ export function instEqual(a, b, lenComp) {
 
 export function instHasClassName(inst, className) {
   const node = findDOMNode(inst);
+  if (!node) { // inst renders null
```

Comment:
That's a good point - https://facebook.github.io/react/docs/react-dom.html#finddomnode makes it seem like it's only `null` or a node. I wonder if then it would be better to use `=== null` specifically so that it blows up if React changes the behavior?